uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/apis/leave-request-api_test.js

```diff
@@ -322,12 +322,17 @@ define([
           });
 
           it('returns validation errors as string', function () {
-            var errorResponse = _.map(mockData.getNotIsValid().values, function (value) {
-              return _.isArray(value) ? value.join('</br>') : JSON.stringify(value);
-            }).join('</br>');
+            var errors = [];
+            _.map(mockData.getNotIsValid().values, function (value) {
```

Comment:
i think this whole thing can be simplified, you simply expect to reject the promise with a flatten array of the mocked errors